src/diagonal.works/b6/api/functions/features.go

```diff
@@ -524,12 +516,12 @@ func listFeature(context *api.Context, id b6.CollectionID) (b6.Collection[any, a
 	return b6.AdaptCollection[any, any](c), nil
 }
 
-func evaluateFeature(context *api.Context, id b6.ExpressionID) (interface{}, error) {
-	e := b6.FindExpressionByID(id, context.World)
+func evaluateFeature(context *api.Context, id b6.FeatureID) (interface{}, error) { // TO(andrew): should this be evaluate expression ??
```

Comment:
I think it should stay as evaluateFeature, because I think evaluateExpression would just take an Expression, not a Feature that contains an expression.

Reply:
aight